src/test/java/com/uber/cadence/workflow/WorkflowTest.java

```diff
@@ -4097,6 +4100,43 @@ public void testGetVersion() {
         "executeActivity customActivity1");
   }
 
+  public static class TestGetVersionWorkflow2Impl implements TestWorkflow1 {
+
+    @Override
+    public String execute(String taskList) {
+      // Test adding a version check in replay code.
+      if (!getVersionExecuted.contains(taskList + "-test_change_2")) {
+        getVersionExecuted.add(taskList + "-test_change_2");
+        Workflow.sleep(Duration.ofHours(1));
+      } else {
+        int version2 = Workflow.getVersion("test_change_2", Workflow.DEFAULT_VERSION, 1);
+        System.out.println("version = " + version2);
```

Comment:
nit: Consider asserting the value of `version2`

Reply:
Removed this print. It's for my own debugging purpose.